content/applications/general/companies/digest_emails.rst

```diff
@@ -69,8 +69,8 @@ Deactivate digest email
 =======================
 
 To manually deactivate an individual digest email, first navigate to :menuselection:`Settings app
---> Statistics section`, and click :guilabel:`Configure Digest Emails`. Then, select the desired
-digest email from the list that should be deactivated.
+--> Emails section`, and click :guilabel:`Configure Digest Emails`. Then, select the desired digest
+email from the list that should be deactivated.
 
 Next, click :guilabel:`DEACTIVATE FOR EVERYONE` to deactivate the digest email for everyone, or
 :guilabel:`UNSUBSCRIBE ME` to remove the logged in user from the mailing list. These buttons are
```

Comment:
I don't see this either of these labels,. Can you double-check and advise, please?

<img width="482" height="247" alt="Image" src="https://github.com/user-attachments/assets/d334c961-4756-455d-ae10-6ef5b01d3476" />